src/Compilers/VisualBasic/Portable/Symbols/PropertySymbol.vb

```diff
@@ -571,12 +573,6 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
-        Private ReadOnly Property IPropertySymbol_IsRequired As Boolean Implements IPropertySymbol.IsRequired
```

Comment:
>Private ReadOnly Property IPropertySymbol_IsRequired As Boolean Implements IPropertySymbol.IsRequired [](http://example.com/codeflow?start=8&length=101)

Please follow the pattern and keep this member. Adjust implementation instead.

Reply:
I'm using the pattern used by this type. As examples, `IsReadOnly` and `IsWriteOnly` do the pattern I did here.

Reply:
See surrounding code for the pattern. It is quite possible that we have a couple of outliers (that should be fixed) and we shouldn't add more deviation. The original code did the right thing for these APIs. let's keep doing the right thing.